src/main.css

```diff
@@ -285,6 +286,60 @@ a {
     transition: 500ms transform cubic-bezier(0.19, 1, 0.22, 1);
 }
 
+.train-popover .occupancy-container { 
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+}
+
+.train-popover .occupancy-details-container { 
+    padding-left: 6px;
+}
+
+.train-popover .occupancy-status-container { 
+    padding-left: 2px;
+    padding-top: 2px;
+    position: relative;
+}
+
+.train-popover .consist-front-icon { 
+    width: 20px;
+    height: 20px;
+    margin-inline: -5px;
+}
+
+.train-popover .occupancy-status-icon { 
+    width: 12px; 
+    height: 12px; 
+    position: 'absolute';
+}
+
+.train-popover .occupancy-status-text {
+    font-size: 8px;
+    padding-left: 1px;
+    color: "#404040";
+}
+
+.train-popover .occupancy-label-container {
+    font-size: 12px;
+    display: flex;
+}
+
+.train-popover .train-carriage {
+    margin-bottom: 1px;
+    height: 30px;
+    width: 10px;
+}
+
+.train-popover .train-carriage.first {
+    border-top-left-radius: 3px;
+    border-top-right-radius: 3px;
+}
+.train-popover .train-carriage.last {
+    border-bottom-left-radius: 3px;
+    border-bottom-right-radius: 3px;
+}
+
```

Comment:
Most of the styles should be in this css and applied by classname, but the dynamic ones are inlined with `style` props for readability